README.md

```diff
@@ -32,16 +32,24 @@ Check out `tutorial.md` to see an example of the different functionalities of th
 You can also run the script directly with the following code in a console:
 
 ```sh
-python vendor_data.py <"-a/--add or -m/--modify">
+python vendor_data.py <-a/--add>

```

Comment:
In this case and below, `-a/--add` and `-m/--modify` are actually literal, so you wouldn't need to add the `<>`.